src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships.cshtml.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Pages
+{
+    [Authorize]
+    public class ApprenticeshipsModel : PageModel
+    {
+        [BindProperty(SupportsGet = true)]
+        public string? ApprenticeshipId { get; set; }
+
+        public void OnGet()
+        {
+        }
+    }
+}
```

Comment:
The route here is expecting a string as part of the path, but when I run the code it just gets into an infinite Loop and never hits this endpoint

Reply:
The page was binding to the wrong model, that's fixed now.